github-dark.css

```diff
@@ -22,6 +22,9 @@
   svg[fill="none"] {
     fill: none;
   }
+  .markdown-body img[data-canonical-src*="svg"], .markdown-body img.emoji {
+		background-color: inherit !important;
+	}
```

Comment:
There's mixed tabs & spaces here

Reply:
I think @the-j0k3r should set up https://editorconfig.org/ in their editor 😉

Reply:
I have editor, I just copied&pasted from issue #983 without running clean

Reply:
My Sublime Text actually does this. If I paste tabs, it will convert to spaces on save. Thought, experience may vary based on editor of course.

Reply:
I'm a little crazy and have render whitespace enabled in mine for this reason.

@silverwind does stylelint's [`indentation`](https://stylelint.io/user-guide/rules/indentation) rule flag these issues or does not play well with our files?